src/exampleWorker.js

```diff
@@ -0,0 +1,229 @@
+/* eslint-env browser */
+
+'use strict';
+
+self.importScripts('../dist/ohm.js', 'utils.js');
+
+// Web Worker that generates examples.
+// communicates with 'exampleGenerationUI.js'
+
+(function(root, initModule) {
+  if (typeof exports === 'object') {
+    module.exports = initModule;
+  } else {
+    initModule(root.ohm, root.utils);
+  }
+})(this, function(ohm, utils) {
+  postMessage('WORKER STARTED');
+  var grammar;
+
+  var examplePieces = {};
+  var generator;
+
+  var overrides = {
+    ident: function(g, ex, syn, args) {
+      return {needHelp: true};
+    },
+    ctor: function(g, ex, syn, args) {
+      return {needHelp: true};
+    },
+    letter: function(g, ex, syn, args) {
+      if (Math.floor(Math.random() * 2) === 0) {
+        return {example: String.fromCharCode(Math.floor(26 * Math.random()) + 'a'.charCodeAt(0))};
+      } else {
+        return {example: String.fromCharCode(Math.floor(26 * Math.random()) + 'A'.charCodeAt(0))};
+      }
+    }
+  };
+
+  self.addEventListener('message', function(e) {
+    postMessage(e.data.name);
+    switch (e.data.name) {
+      case 'initialize':
+        /* eslint-disable no-eval */
+        grammar = ohm.makeRecipe(eval(e.data.recipe));
```

Comment:
Not anymore. Marko changed it a few weeks ago to be JSON...and in fact, I think you can just pass the string to `makeRecipe` and it will parse it for you.